sys-utils/libcmt/Makefile

```diff
@@ -76,10 +77,13 @@ $(libcmt_OBJ): $(libcmt_OBJDIR)/%.o: %.c
 $(libcmt_LIB): $(libcmt_OBJ)
 	$(TARGET_AR) rcs $@ $^
 
-libcmt: $(libcmt_LIB)
-install: $(libcmt_LIB)
+$(libcmt_SO): $(libcmt_OBJ)
+	$(TARGET_CC) -shared -o $@ $^
+
+libcmt: $(libcmt_LIB) $(libcmt_SO)
+install: $(libcmt_LIB) $(libcmt_SO)
```

Comment:
In spirit of minimal rootfs, I think it's good idea to split in two packages:
- libcmt-dev - contains .h and .a files
- libcmt - contains .so files

And they are not dependent on each other. For instance Alpine split some packages like this, to be minimal.

So we would need to add a `install-dev` target.

Reply:
Nice, I'll create a separate issue for that. @vfusco mentioned that we'll do a minor release soon with the `.so`.

Reply:
here: https://github.com/cartesi/machine-emulator-tools/issues/59